### PR TITLE
Send welcome SMS

### PR DESCRIPTION
**Pull request by gsvr**
Directly to Outbound API on Message Sender

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: registrations/tasks.py [hunk 1]
```diff
@@ -269,6 +270,40 @@ def create_subscriptionrequests(self, registration):
...
+        # Insert the mother's health_id in the SMS
+        mother = utils.get_identity(registration.mother_id)
+        if mother["details"]["health_id"]:
+            sms = sms.replace(
+                '[health_id]', str(mother["details"]["health_id"]))
+        else:
+            # TODO: raise exception?
+            pass
+
+        print(sms)
```
Comment (imsickofmaps) [inline registrations/tasks.py]: stray print

File: registrations/tasks.py [hunk 2]
```diff
@@ -269,6 +270,40 @@ def create_subscriptionrequests(self, registration):
...
+                sms = settings.HOUSEHOLD_HW_WELCOME_TEXT_UG_ENG
+            else:
+                sms = settings.HOUSEHOLD_PUBLIC_WELCOME_TEXT_UG_ENG
+        # Insert the mother's health_id in the SMS
+        mother = utils.get_identity(registration.mother_id)
+        if mother["details"]["health_id"]:
+            sms = sms.replace(
+                '[health_id]', str(mother["details"]["health_id"]))
+        else:
+            # TODO: raise exception?
```
Comment (imsickofmaps) [inline registrations/tasks.py]: Please open an Issue for this. This is part of the bigger narrative of "what if the ID generator fails".

**Comment by imsickofmaps** [comment]
Minor tweak then :+1: when travis happy

**Pull request merged**